### DISCUSSION TITLE
4.12 vulnerability of Junit

### DISCUSSION THREAD

**Discussion by xuyf1001** [apache/skywalking#11254] [General]
Now I need to fix the 4.12 vulnerability of Junit on version 8.9 of Skywalking, but I don't know which parts of SkyWalking are affected by this vulnerability, because there are so many packages with Junit tests, do you have any good suggestions？

**Comment by kezhenxu94** [comment]
JUnit is test phase dependency, SkyWalking runtime doesn't contains any of those dependencies, why would you bother to do such "fix"

**Reply by xuyf1001** [reply]
We are doing Euler's localized components, one of which is skywalking, our self-compiled docker image has a Junit vulnerability after scanning.The leader let me to fix, but I looked at the source code of skyWalking and apm.tar.gz and found that this vulnerability does not seem to have an impact, so I come to confirm it. I have already got the answer. Thank you.
